src/Mod/CAM/Path/Base/Language.py

```diff
@@ -37,7 +37,7 @@ class Instruction(object):
 
     def __init__(self, begin, cmd, param=None):
         self.begin = begin
-        if type(cmd) == Path.Command:
+        if type(cmd) is Path.Command:
```

Comment:
If you're going to change this line, may as well make it idiomatic and use `isinstance()`.